workspaces/global-header/plugins/global-header/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@red-hat-developer-hub/backstage-plugin-global-header",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.16.1",
+    "@backstage/core-plugin-api": "^1.10.1",
+    "@backstage/plugin-catalog-react": "^1.15.0",
+    "@backstage/theme": "^0.6.2",
+    "@mui/base": "^5.0.0-beta.66",
```

Comment:
Hi @christoph-jerolimov , is it a big problem to use `@mui/base` components? Backstage will change their frontend into a headless style, check this out
https://github.com/backstage/backstage/issues/27726

And there are implemented dropdown menu close/open control, which is very handy. I can take out of `@mui/base` and use only `@mui/material`, just it will take some time and eventually we will switch to a headless style anyways. What do you think?